TestApp/android/app/src/main/java/com/testapp/MainApplication.java

```diff
@@ -37,19 +26,14 @@ public boolean getUseDeveloperSupport() {
 
         @Override
         protected List<ReactPackage> getPackages() {
-            return Arrays.asList(
-                    new MainReactPackage(),
-                    new RNGestureHandlerPackage(),
-                    new TestAppNativePackage(getApplication()),
-                    new AppCenterReactNativeAuthPackage(getApplication()),
-                    new AppCenterReactNativePushPackage(getApplication()),
-                    new AppCenterReactNativeCrashesPackage(getApplication(), getResources().getString(R.string.appCenterCrashes_whenToSendCrashes)),
-                    new AppCenterReactNativeAnalyticsPackage(getApplication(), getResources().getString(R.string.appCenterAnalytics_whenToEnableAnalytics)),
-                    new AppCenterReactNativePackage(getApplication()),
-                    new RNFSPackage(),
-                    new ImagePickerPackage(),
-                    new AsyncStoragePackage()
-            );
+            @SuppressWarnings("UnnecessaryLocalVariable")
+            List<ReactPackage> packages = new PackageList(this).getPackages();
```

Comment:
inline variable

Reply:
This is code generated by `react-native init`, so probably okay to keep it.